source/developer/mobile-developer-setup.rst

```diff
@@ -0,0 +1,147 @@
+..  _mobile-developer-setup:
+
+Mobile Developer Machine Setup
+==================================
+
+The following instructions apply to new unreleased mobile apps for iOS and Android built in React Native. They are intended to release in Beta at the end of March 2017. Source code can be found at: https://github.com/mattermost/mattermost-mobile
```

Comment:
@JeffSchering will this autolink once it's on docs site, or do we need to format it as a link?